cmd/influx_stress/influx_stress.go

```diff
@@ -3,125 +3,55 @@ package main
 import (
 	"flag"
 	"fmt"
-	"runtime"
-	"sort"
-	"time"
+	"os"
+	"runtime/pprof"
 
 	"github.com/influxdb/influxdb/stress"
 )
 
 var (
-	batchSize     = flag.Int("batchsize", 0, "number of points per batch")
-	concurrency   = flag.Int("concurrency", 0, "number of simultaneous writes to run")
-	batchInterval = flag.Duration("batchinterval", 0*time.Second, "duration between batches")
-	database      = flag.String("database", "", "name of database")
-	address       = flag.String("addr", "", "IP address and port of database (e.g., localhost:8086)")
-	precision     = flag.String("precision", "", "The precision that points in the database will be with")
-	test          = flag.String("test", "", "The stress test file")
+	//database  = flag.String("database", "", "name of database")
```

Comment:
Not necessarily.

My idea is that you'll be allowed to modify database and address. Anything else that you'd like to change requires that you touch the config file.